controllers/component_controller_test.go

```diff
@@ -359,12 +359,16 @@ var _ = Describe("Component controller", func() {
 			Eventually(func() bool {
 				k8sClient.Get(context.Background(), hasCompLookupKey, createdHasComp)
 				return len(createdHasComp.Status.Conditions) > 0
-			}, timeout, interval).Should(BeTrue())
+			}, 5*timeout, interval).Should(BeTrue())
```

Comment:
any reason this one uses `5*timeout`? and the other one uses `4*timeout`?

Reply:
Nope, no reason in particular. Will update this

Reply:
Updated.